### ISSUE TITLE
push notifications

### ISSUE THREAD

**Issue by VictorHom** [AndroidHuddle/Huddle#22]
how to set that up

**Comment by badhrijagan06** [comment]
I will work on this

**Comment by badhrijagan06** [comment]
Since we are not using parse's session management, I have a non-orthodox way of sending unicast messages.. Just to make sure we always have a working front end and back end, I have hardcorded a push request in Dashboard.. So you should see a notification whenever Dashboard is loaded.

**Comment by aliciaphes** [comment]
Is there any way we can integrate [this](https://guides.codepath.com/android/Notifications) with those parse messages?

**Comment by badhrijagan06** [comment]
yes Alicia.. under the hood it calls into Notification  manager.. I am currently implementing a custom listener.. I am positive that we should be able to customize the notification content once I have that in place.. Are you thinking along the lines of notifications for event invites ? That can be easily done with the logic that I already have in place..
